developer-docs-site/docs/tutorials/first-move-module.md

```diff
@@ -38,6 +38,9 @@ For this tutorial, will be focusing on `hello_blockchain/src` and re-using the `
 
 You can find the rust project [here](https://github.com/aptos-labs/aptos-core/tree/main/developer-docs-site/static/examples/rust)
 
+The key is that you need have two terminals open at the same time:  One is Move terminal, the other is Python terminal, and the sequence of events of switching these two terminals is very important.
+
```

Comment:
I suggest we remove the above passage. We should rely on the intelligence of the reader to figure this out themselves. Or else we will have to tell them a whole lot more low-level things to be consistent in our docs.